tools/get_styles_colors.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+use Sabberworm\CSS\Property\Selector;
+use Sabberworm\CSS\Rule\Rule;
+use Sabberworm\CSS\RuleSet\DeclarationBlock;
+use Sabberworm\CSS\Value\Color;
+use Symfony\Component\Finder\Finder;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$finder = new Finder();
+$finder
+    ->in(__DIR__ . '/../styles/')
+    ->name('*.css')
+    ->files()
+;
+
+$backgroundColors = array();
+
+/** @var \Symfony\Component\Finder\SplFileInfo $file */
+foreach ($finder->getIterator() as $file) {
+    $color = '#ffffff';
+    $cssParser = new Sabberworm\CSS\Parser($file->getContents());
+    $cssDocument = $cssParser->parse();
+
+    /** @var DeclarationBlock $ruleSet */
+    foreach ($cssDocument->getAllRuleSets() as $ruleSet) {
+        /** @var Selector $selector */
+        foreach ($ruleSet->getSelectors() as $selector) {
+            if ($selector->getSelector() === '.hljs') {
+                $bgColor = $ruleSet->getRules('background');
+
+                if (empty($bgColor)) {
+                    $bgColor = $ruleSet->getRules('background-color');
+                }
+
+                /** @var Rule $value */
+                foreach ($bgColor as $value) {
+                    if ($value->getValue() instanceof Color) {
+                        $color = preg_replace('/background(?:-color)?:\s|;/', '', (string) $value);
```

Comment:
I think it would be preferable to use `\Sabberworm\CSS\Value\Color::getColor()` here so that the underlying RGB colors can be returned (as an array), as opposed to a hexadecimal representation (as a string).

```suggestion
                        $color = $value->getValue()->getColor();
```

This will avoid the need to re-parse the hex string, and the values can be passed straight into an `rgb()` function.

Reply:
I like this much better. I'm returning an array of numerical RGB values now instead.